finagle-http/src/test/scala/com/twitter/finagle/http/filter/StatsFilterTest.scala

```diff
@@ -12,34 +12,46 @@ import org.mockito.Mockito.{spy,verify,times}
 @RunWith(classOf[JUnitRunner])
 class StatsFilterTest extends FunSuite {
 
+  val service = new Service[Request, Response] {
+    def apply(request: Request): Future[Response] = {
+      val response = request.response
+      response.statusCode = 404
+      response.write("hello")
+      Future.value(response)
+    }
+  }
+
   test("increment stats") {
     val receiver = spy(new InMemoryStatsReceiver)
 
-    val filter = new StatsFilter(receiver) andThen new Service[Request, Response] {
-      def apply(request: Request): Future[Response] = {
-        val response = request.response
-        response.statusCode = 404
-        response.write("hello")
-        Future.value(response)
-      }
+    val filter = new StatsFilter(receiver) andThen service
+
+    Time.withCurrentTimeFrozen { _ =>
+      Await.result(filter(Request()))
```

Comment:
ah, when I wrote "we no longer use Await.result" I meant unbounded Await.result.